airbyte-cdk/bulk/toolkits/load-object-storage/src/main/kotlin/io/airbyte/cdk/load/file/object_storage/ObjectStoragePathFactory.kt

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.file.object_storage
+
+import io.airbyte.cdk.load.command.DestinationStream
+import io.airbyte.cdk.load.command.object_storage.ObjectStoragePathConfigurationProvider
+import io.airbyte.cdk.load.file.TimeProvider
+import io.micronaut.context.annotation.Secondary
+import jakarta.inject.Singleton
+import java.nio.file.Path
+import java.nio.file.Paths
+import java.util.*
+import org.joda.time.DateTime
```

Comment:
can this just be java.time.Instant? (afaik joda time shouldn't really be a thing in jdk>=8)

Reply:
I am old brain